hw/xbox/nv2a/nv2a_regs.h

```diff
@@ -1282,6 +1286,9 @@
 #define NV_IGRAPH_XF_LTC1_L6                         0x12
 #define NV_IGRAPH_XF_LTC1_L7                         0x13
 
+#define RDI_INDEX_VTX_CONSTANTS0                     0x17
+#define RDI_INDEX_VTX_CONSTANTS1                     0xCC
```

Comment:
These appear to be names given by MS.

nvidia seems to refer to `RDI_INDEX_VTX_CONSTANTS0` by `RDI_RAMSEL_XL_XFCTX`. However, `RDI_INDEX_VTX_CONSTANTS1` doesn't seem to have an nvidia name.

I suggest we update these names when they are named in nouveau.

We can still use nvidia names (if we find them somewhere) in other instances - but I'd rather not mix MS and nvidia names for closely related registers.

Reply:
If we mix name conventions here, we should probably leave a comment here indicating what/why this was done and what the alternative naming is. Also see comment above with regard to `CONSTANTS1`.

Reply:
I'll move my comment into the code then, although we have other instances were it appears that non-nvidia names have been chosen, with no hints (like most `NV2A_*` defines right below).